<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DslamAddNewVlanDialog</class>
 <widget class="QDialog" name="DslamAddNewVlanDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>501</width>
    <height>350</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Tahoma</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>DSLAM Add New VLAN</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../../image.qrc">
    <normaloff>:/Image/add.png</normaloff>:/Image/add.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="dslamLbl">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(85, 0, 127);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="servicesGB">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>200</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>420</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Services New VLAN</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_28">
        <property name="spacing">
         <number>50</number>
        </property>
        <item>
         <widget class="QLabel" name="label_44">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Tahoma</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 127);</string>
          </property>
          <property name="text">
           <string>HSI PPPOE</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_45">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>2000</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Tahoma</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 127);</string>
          </property>
          <property name="text">
           <string> VLAN</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="hsiSB">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Tahoma</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(85, 0, 127);</string>
          </property>
          <property name="minimum">
           <number>500</number>
          </property>
          <property name="maximum">
           <number>599</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_29">
        <property name="spacing">
         <number>50</number>
        </property>
        <item>
         <widget class="QLabel" name="label_46">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Tahoma</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 127);</string>
          </property>
          <property name="text">
           <string>INET PPPOE</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_47">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>2000</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Tahoma</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 127);</string>
          </property>
          <property name="text">
           <string> VLAN</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="inetSB">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Tahoma</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(85, 0, 127);</string>
          </property>
          <property name="minimum">
           <number>600</number>
          </property>
          <property name="maximum">
           <number>699</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_30">
        <property name="spacing">
         <number>50</number>
        </property>
        <item>
         <widget class="QLabel" name="label_48">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Tahoma</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 127);</string>
          </property>
          <property name="text">
           <string>SINET PPPOE</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_49">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>2000</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Tahoma</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 127);</string>
          </property>
          <property name="text">
           <string> VLAN</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sinetSB">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Tahoma</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(85, 0, 127);</string>
          </property>
          <property name="minimum">
           <number>700</number>
          </property>
          <property name="maximum">
           <number>799</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_31">
        <property name="spacing">
         <number>50</number>
        </property>
        <item>
         <widget class="QLabel" name="label_50">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Tahoma</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 127);</string>
          </property>
          <property name="text">
           <string>VOIP PPPOE</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_51">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>2000</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Tahoma</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 127);</string>
          </property>
          <property name="text">
           <string> VLAN</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="voipSB">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Tahoma</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(85, 0, 127);</string>
          </property>
          <property name="minimum">
           <number>800</number>
          </property>
          <property name="maximum">
           <number>899</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="infoLbl">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(85, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>17</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="confirmChB">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(85, 85, 0);</string>
       </property>
       <property name="text">
        <string>Confirm</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(92, 53, 102);</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../image.qrc">
         <normaloff>:/Image/remove.png</normaloff>:/Image/remove.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="okBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(92, 53, 102);</string>
       </property>
       <property name="text">
        <string>Add VLANs</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../image.qrc">
         <normaloff>:/Image/add.png</normaloff>:/Image/add.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../image.qrc"/>
 </resources>
 <connections/>
</ui>
